rewrite-java-11/src/main/java/org/openrewrite/java/Java11ParserVisitor.java

```diff
@@ -1729,6 +1730,16 @@ private JavaType type(@Nullable com.sun.tools.javac.code.Type type, List<Symbol>
                     if (sym.owner instanceof Symbol.ClassSymbol) {
                         owner = TypeUtils.asFullyQualified(type(sym.owner.type, stackWithSym));
                     }
+
+
+                    // Add annotation information to JavaType.Class
+                    List<JavaType.Class> annotations = new ArrayList<>();
+                    if(sym.getMetadata() != null && sym.getMetadata().getDeclarationAttributes() != null ) {
```

Comment:
Looking at the source for symbol `sym.getMetadata()` has a warning that is it only used for debugging: 

https://github.com/openjdk/jdk/blob/jdk-11%2B28/src/jdk.compiler/share/classes/com/sun/tools/javac/code/Symbol.java#L265-L268

you may want to use: `sym.getDeclarationAttributes()` instead.


You may also want to check `sym. annotationsPendingCompletion()`, as I suspect if a symbol that has not been completed, may not have its metadata.

Reply:
Looking at the compiler code, I think `sym.getMetadata()` is correct. As the compiler appends attributes, it inits the metadata. If no attribute is ever appended, it will be null, and the null check catches this.